app/routes.js

```diff
@@ -1,26 +1,31 @@
 import express from 'express';
 import { getPublicSolutionById } from './view-solution/controller';
 import { getSolutionListPageContext } from './solutions-list/controller';
+import { getBrowseSolutionsPageContext } from './browse-solutions/context';
+import { getHomepageContext } from './homepage/context';
+import footerLinks from './manifest.json';
 
 const router = express.Router();
 
-router.get('/', async (req, res) => {
-  res.render('homepage/template.njk', {});
+router.get('/', (req, res) => {
+  const context = { ...getHomepageContext(), ...footerLinks };
+  res.render('homepage/template.njk', context);
 });
 
-router.get('/solutions', async (req, res) => {
-  res.render('browse-solutions/template.njk', {});
+router.get('/solutions', (req, res) => {
+  const context = { ...getBrowseSolutionsPageContext(), ...footerLinks };
+  res.render('browse-solutions/template.njk', context);
 });
 
 router.get('/solutions/:filterType', async (req, res) => {
   const { filterType } = req.params;
-  const context = await getSolutionListPageContext(filterType);
+  const context = { ...await getSolutionListPageContext(filterType), ...footerLinks };
   res.render('solutions-list/template.njk', context);
 });
 
 router.get('/solutions/:filterType/:solutionId', async (req, res) => {
   const { solutionId } = req.params;
-  const context = await getPublicSolutionById(solutionId);
+  const context = { ...await getPublicSolutionById(solutionId), ...footerLinks };
```

Comment:
This is all perfect. I'm just not sure about having to pass in `footerLinks` each time to create the context.  Can we leave the footerLinks hardcoded as they was before and revist this later?

Reply:
yeah I wasn't sure about it either but I kept it because it made the tests easier but can change back

Reply:
yeah I would change it back. But other than that it looks good.

Reply:
👍 going to merge now